api/tests/acceptance/application/campaign-controller_test.js

```diff
@@ -1278,4 +1278,70 @@ describe('Acceptance | API | Campaign Controller', function () {
       expect(response.statusCode).to.equal(400);
     });
   });
+
+  describe('PATCH /api/campaigns/{id}', function () {
+    it('should return 200 when user is admin or owner of the campaign', async function () {
+      // given
+      const organization = databaseBuilder.factory.buildOrganization();
+      const campaign = databaseBuilder.factory.buildAssessmentCampaign({ organizationId: organization.id });
+      const userId = databaseBuilder.factory.buildUser().id;
+      databaseBuilder.factory.buildMembership({ userId, organizationRole: 'ADMIN', organizationId: organization.id });
+      await databaseBuilder.commit();
+
+      // when
+      const response = await server.inject({
+        method: 'PATCH',
+        url: `/api/campaigns/${campaign.id}`,
+        headers: { authorization: generateValidRequestAuthorizationHeader(userId) },
+        payload: {
+          data: {
+            type: 'campaigns',
+            attributes: {
+              name: 'toto',
+              title: null,
+              'custom-landing-page-text': 'toto',
+              'owner-id': userId,
+            },
+          },
+        },
+      });
+
+      // then
+      expect(response.statusCode).to.equal(200);
+    });
+
+    it('should return 403 when user is not an admin or owner of the campaign', async function () {
```

Comment:
Il a une 403 car il n'est pas admin ET il n'est pas propriétaire. Je suggère de renommer le test en `should return 403 when user is not an admin and is not the campaign owner`

Reply:
corrigé